lua/niknaks/modules/sh_color_extended.lua

```diff
@@ -152,4 +152,42 @@ local string_format, string_sub = string.format, string.sub
 	---@return boolean
 	function COLOR:IsDark()
 		return ColorToLuminance(self) < 127.5
-	end
\ No newline at end of file
+	end
+
+-- ColorRGBExp32
+	local gamma = 2.2
+	local overbrightFactor = 0.5
+
+	-- convert texture to linear 0..1 value
+	local function TexLightToLinear( col, exponent )
+		return col * ( ( 2 ^ exponent ) / 255  )
+	end
+
+	-- linear (0..4) to screen corrected vertex space (0..1?)
+	local function LinearToVertexLight( col )
+		return overbrightFactor * ( col ^ ( 1 / gamma ) )
+	end
+
+	-- https://github.com/ValveSoftware/source-sdk-2013/blob/master/sp/src/utils/vrad/lightmap.cpp#L3551
```

Comment:
https://github.com/ValveSoftware/source-sdk-2013/blob/master/sp/src/utils/vrad/lightmap.cpp#L3551

Reply:
Feel like Source might do so odd things in the background. My guess is a shader or some value on the render-end.
I'll add it to the todo list to look over.